### ISSUE TITLE
The reason for val_acc=0.00

### ISSUE THREAD

**Issue by sarkarghya** [sarkarghya/tfjs-node-pose-trainer#10]
Here is an example output
```
Epoch 1 / 1
eta=0.0 ===========================================================================> 
4568ms 87854us/step - acc=0.192 loss=1.58 val_acc=0.00 val_loss=1.72 
```

The reason for val_acc=0.00 is not clear and is probably because of some data issues. The reason can be as follows

1. **Data Issues:**
   - **Data Splitting:** Ensure that your training and validation datasets are correctly split. If the validation set is empty or too small, the accuracy might appear as 0.
   - **Labeling:** Check if the labels in your validation set are correctly assigned. If labels are not properly matched with the data, it can lead to 0 accuracy.

2. **Model Issues:**
   - **Model Complexity:** The model might be too complex for the given data, leading to overfitting on the training set and poor generalization to the validation set. Consider simplifying your model or using regularization techniques.
   - **Learning Rate:** The learning rate might be too high, causing the model to overshoot the optimal weights. Try reducing the learning rate and observing the changes in performance.

3. **Loss Function:**
   - **Incorrect Loss Function:** Ensure that you are using an appropriate loss function for your problem. Using a regression loss function for a classification problem, or vice versa, can lead to unexpected results.
